src/wallet/spend.h

```diff
@@ -54,24 +51,17 @@ class COutput
     /** Whether the transaction containing this output is sent from the owning wallet */
     bool m_from_me;
 
-    COutput(const CWallet& wallet, const CWalletTx& wtx, int iIn, int depth, bool spendable, bool solvable, bool safe, int64_t time, bool from_me, bool use_max_sig_in = false)
+    COutput(const CWallet& wallet, const CWalletTx& wtx, int iIn, int depth, int input_bytes, bool spendable, bool solvable, bool safe, int64_t time, bool from_me)
```

Comment:
In commit "wallet: Provide input bytes to COutput" (353d7bf8e7c800aa19cbcc52fe0636667c754cec)

Nice to drop the use_max_sig argument here, but this should definitely drop the CWallet argument too, which is no longer needed here

EDIT: I see it is dropped in later commit